src/back/spv/writer.rs

```diff
@@ -1109,8 +1109,11 @@ impl Writer {
                 "64-bit integer storage image format",
                 &[spirv::Capability::Int64ImageEXT],
             ),
-            If::Unknown
-            | If::Rgba32f
+            If::Unknown => self.require_any(
+                "Unknown storage image format",
+                &[spirv::Capability::StorageImageWriteWithoutFormat],
```

Comment:
Can we instead do this in the `write_image_store` fn? It should be more accurate since the `Unknown` format is also used for sampled images.

```rust
match *self.fun_info[image].ty.inner_with(&self.ir_module.types) {
    crate::TypeInner::Image {
        class:
            crate::ImageClass::Storage {
                format: crate::StorageFormat::Bgra8Unorm,
                ..
            },
        ..
    } => self.writer.require_any(
        "Bgra8Unorm storage write",
        &[spirv::Capability::StorageImageWriteWithoutFormat],
    )?,
    _ => {}
}
```